spec/rails_helper.rb

```diff
@@ -46,6 +46,9 @@
 
   config.before(:suite) do
     DatabaseCleaner.clean_with(:truncation)
+    # Seems cleaner to just put this here than track exactly which tests need the seed data.
+    # The Languages dependency at least is pretty fundamental
+    seed_data
```

Comment:
It seems to be working as I expected.  And the full set of tests were running fine in my branch.  The way that I understood the 
  DatabaseCleaner.strategy = :transaction
setting is that it's not actually truncating before test, but relying on the transaction rollback.  
(I was quite puzzled at first when I was trying to debug the unit test behavior because I wasn't able to see any of the intermediate database state while pausing execution in the middle of the test.  And that is how I realized there was a transactional approach being used before I noticed the DatabaseCleaner configuration)

I personally think that the seed data should be accepted as legitimate system configuration state which can be relied up to exist within the unit tests.  It feels an unnecessary effort and added complexity to try to write all of the unit tests without its existence.

@linesarefuzzy  Do you have a perspective here?